services/autoscaling/tests/unit/test_modules_auto_scaling_computational.py

```diff
@@ -79,7 +79,6 @@ def minimal_configuration(
     local_dask_scheduler_server_envs: EnvVarsDict,
     mocked_ec2_instances_envs: EnvVarsDict,
     disabled_rabbitmq: None,
-    disabled_opentelemetry: None,
```

Comment:
question: here you are enabling telemetry in the tests?
how does that even work if there is no telemtry collector?
and subsequent question:
how does this work in the other services? are we having errors all over the place? or is this handled?

Reply:
This was not working.

I have disabled telemetry in the root contest.py fixture that loads the environment.


I have also applied the same pattern in all the other services and now opentelemetry setup is disabled during tests.